core/lib/generators/refinery/cms/cms_generator.rb

```diff
@@ -219,10 +219,11 @@ def mount!
         mount = %Q{
   # This line mounts Refinery's routes at the root of your application.
   # This means, any requests to the root URL of your application will go to Refinery::PagesController#home.
-  # If you would like to change where this extension is mounted, simply change the :at option to something different.
+  # If you would like to change where this extension is mounted, simply change the
+  # configuration option `mounted_path` to something different in config/initializers/refinery/core.rb
   #
   # We ask that you don't use the :as option here, as Refinery relies on it being the default of "refinery"
```

Comment:
Yes, unless you can find a way to work around it. We reference `refinery.path_helper` everywhere.